### PR TITLE
Directory features

### PR DESCRIPTION
**Pull request by Mark-de-Haan**

### CODE REVIEW

**Comment by ConnorStroomberg** [comment]
- 008/009  open one or two top most facets to single to user there are interface elements on left side.

File: src/components/SearchBox.vue [hunk 1]
```diff
@@ -2,14 +2,21 @@
   <div class="search-box-container">
     <div class="input-group search-input-container">
       <input type="text" class="form-control search-input" v-model.lazy="search"
-             placeholder="Search through biobanks and collections...">
+             placeholder="Search biobanks by name">
```
Comment (ConnorStroomberg) [inline src/components/SearchBox.vue]: could we use i18N to make this dynamic ?

File: src/components/filters/MaterialFilters.vue [hunk 2]
```diff
@@ -37,7 +37,7 @@
     name: 'material-filters',
     data () {
       return {
-        collapsed: true,
+        collapsed: false,
```
Comment (ConnorStroomberg) [inline src/components/filters/MaterialFilters.vue]: does not seem to work ?

File: src/components/filters/ActiveFilterList.vue [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+</style>
+
+<script>
+  import { UPDATE_FILTER, RESET_FILTERS } from '../../store/mutations'
+  import { mapGetters } from 'vuex'
+
+  export default {
+    name: 'active-filter-list',
+    methods: {
+      removeFilter (filterType, selectedFilterId) {
```
Comment (ConnorStroomberg) [inline src/components/filters/ActiveFilterList.vue]: add unit test

File: src/components/filters/FilterContainer.vue [hunk 4]
```diff
@@ -13,7 +13,7 @@
   }
 
   .filter-header {
-    background-color: #e4e4e4;
+    background-color: #f5f5f5;
```
Comment (ConnorStroomberg) [inline src/components/filters/FilterContainer.vue]: magic number ? add comment what this hard coded color does

File: src/components/SearchBox.vue (hunk 1 above)
Comment (Mark-de-Haan) [reply src/components/SearchBox.vue]: I think that if we do so, we should do it of everything.
We could always do it when ever a feature request for more configuration / internationalization comes along

File: src/components/filters/FilterContainer.vue (hunk 4 above)
Comment (Mark-de-Haan) [reply src/components/filters/FilterContainer.vue]: It is now the same color gray as the biobank headers.
But added a comment mentioning this

**Pull request merged**